doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -65,27 +65,27 @@ Setting properties of a function's arguments
          .. exn:: The & modifier may only occur once.
             :undocumented:
 
-      :n:`( ... ) {? % @scope }`
+      :n:`( ... ) {* % @scope }`
          :n:`(@name__1 @name__2 ...)%@scope` is shorthand for :n:`@name__1%@scope @name__2%@scope ...`
 
-      :n:`[ ... ] {? % @scope }`
+      :n:`[ ... ] {* % @scope }`
          declares the enclosed names as implicit, non-maximally inserted.
          :n:`[@name__1 @name__2 ... ]%@scope` is equivalent to :n:`[@name__1]%@scope [@name__2]%@scope ...`
 
-      :n:`%{ ... %} {? % @scope }`
+      :n:`%{ ... %} {* % @scope }`
```

Comment:
These text descriptions need revision to reflect that there may be multiple scopes. is just `@name__1%scope__1%scope__2 ...`  Does "add_inside" just mean "before" (or "after")?